docs/development/hot-reloading-web.md

```diff
@@ -5,9 +5,10 @@
 > and hit save, after a short moment the web page will reload and your added
 > HTML tag is visable instantly.
 
-In our `docker-compose.yml` to run Wharf locally with we've disabled the `web`
-service by default because we developers find ourselves usually editing the
-frontend and backend from time to time.
+In our `docker-compose.override.yml`/`docker-compose.build.yml` file to run
+Wharf locally with we've disabled the `web` service by default because we
+developers find ourselves usually editing the frontend and backend from
+time to time.
```

Comment:
I find the grammar and language a bit off in this section. How about something like this?
```suggestion
In our local docker config: `docker-compose.override.yml` & `docker-compose.build.yml` we have disabled the `web` service by default because we find it more convenient to run it outside the docker environment.
```

Reply:
I like it!

Reply:
Though they are not different files. That's why I used a slash instead of ampersand

Reply:
I've taken your suggestion and modified it a little